recipes/debugging/vs-code-debug.md

```diff
@@ -91,11 +91,11 @@ To install GDB or LLDB according with your OS use one of the following commands:
 
 ![debugging](https://i.imgur.com/GsGT1h0.png)
 
-## Trips and Tricks for debugging Crystal applications
+## TTips and Tricks for debugging Crystal applications
 
-> DISCLAIMER: LLDB doesn't show variables data for crystal applications yet, see [issue #4457](https://github.com/crystal-lang/crystal/issues/4457).
+> DISCLAIMER: LLDB does not show variable data for crystal applications yet, see [issue #4457](https://github.com/crystal-lang/crystal/issues/4457).
```

Comment:
LLDB does not show data for variables in crystal applications yet

(I know I suggested the wording, but re-reading it, I think this newest line is better.)